packages/button/src/LionButton.js

```diff
@@ -229,6 +230,11 @@ export class LionButton extends DisabledWithTabIndexMixin(SlotMixin(LitElement))
     }
   }
 
+  // eslint-disable-next-line class-methods-use-this
+  __nativeButtonHandler(event) {
+    event.stopPropagation();
```

Comment:
So implicit form submission still works like this?

Reply:
Yeah, because it is only stopping the propagation of the event, it does not prevent the default action (in this case the submit action).

> The Event interface's preventDefault() method tells the user agent that if the event does not get explicitly handled, its default action should not be taken as it normally would be. [MDN preventDefault docs](https://developer.mozilla.org/en-US/docs/Web/API/Event/preventDefault)